gitalias.txt

```diff
@@ -313,6 +316,9 @@
   # This is a slightly modified version
   fixup = "!f() { TARGET=$(git rev-parse \"$1\"); git commit --fixup=$TARGET && GIT_EDITOR=true git rebase --interactive --autosquash $TARGET~; }; f"
 
+  # Shorthand to stash/rebase/unstash
+  fix = !"f(){ STASH=$(git diff --quiet || git stash);git rebase -i \"${@:-HEAD^^}\";[ -n \"$STASH\" ] && git stash pop;};f"
+
```

Comment:
Isn't stash/rebase/unstash just `git rebase --autostash`?

Reply:
I did not know about `--autostash`. That's why I like this repository, I learn a lot, thank you!